core/iwasm/aot/aot_loader.c

```diff
@@ -231,7 +293,11 @@ const_str_set_insert(const uint8 *str, int32 len, AOTModule *module,
         return NULL;
     }
 
-    bh_memcpy_s(c_str, (uint32)(len + 1), str, (uint32)len);
+    if(is_vram_align_word) {
+        bh_memcpy_aw(c_str, (uint32)(len + 1), str, (uint32)len);
+    } else {
+        bh_memcpy_s(c_str, (uint32)(len + 1), str, (uint32)len);
+    }
```

Comment:
since the is_vram_align_word is judged already by the macro read_string when it is defined:
`#define read_string(p, p_end, str)                                \
    do {                                                          \
        if (!(str = load_string((uint8 **)&p, p_end, module,      \
                                is_load_from_file_buf, **true**,      \
                                error_buf, error_buf_size)))      \
            goto fail;                                            \
    } while (0)`
and read_string has been wrappered by "defined(BUILD_TARGET_XTENSA) && defined(...)", shall we need re-wrap again?

Reply:
Yes, had better wrap again here, if not, the bh_memcpy_aw will be linked, which introduces the footprint.